rss-client/src/app/User/Admin/admin-page/admin-page.component.html

```diff
@@ -1,61 +1,69 @@
 <!-- admin check -->
 <div *ngIf="user.admin">
-    <!-- main container -->
-    <div [ngSwitch]="view">
-        <!-- the same view can be shown in more than one case -->
-        <ng-template [ngSwitchCase]="'quiz'">
-            <admin-quiz (navigateEvent)="navigateTo($event)"></admin-quiz>
-        </ng-template>
-        <ng-template [ngSwitchCase]="'inventory'">
-            <admin-inventory (navigateEvent)="navigateTo($event)"></admin-inventory>
-        </ng-template>
-        <!--default case when there are no matches -->
-        <ng-template [ngSwitchDefault]>
+  <!-- main container -->
+  <div [ngSwitch]="view">
+    <!-- the same view can be shown in more than one case -->
+    <ng-template [ngSwitchCase]="'quiz'">
+      <admin-quiz (navigateEvent)="navigateTo($event)"></admin-quiz>
+    </ng-template>
+    <ng-template [ngSwitchCase]="'inventory'">
+      <admin-inventory (navigateEvent)="navigateTo($event)"></admin-inventory>
+    </ng-template>
+    <!--default case when there are no matches -->
+    <ng-template [ngSwitchDefault]>
+      <!-- <div class="container">
+      <!- quizzes 
+      <div class="border container">
+        <button (click)="navigateTo('quiz')">go here</button>
+        some info about admin quiz stuff
+      </div>
 
-            <div class="row justify-content-center">
-                <div class="col-4">
-                    <div class="card" style="width: 18rem;">
-                        <img class="card-img-top" src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQGOq8zcxHug0hBzXDNHOyUGQB9FJOHZsFiWw&usqp=CAU" alt="Quiz card image" />
-                        <div class="card-body">
-                            <h5 class="card-title">Quizzes</h5>
-                            <p class="card-text">
-                                Here you can create new quizzes or edit existing ones.
-                            </p>
-                            <button class="btn btn-primary" (click)="navigateTo('quiz')">
+      <!- inventory 
+      <div class="border container">
+        <button (click)="navigateTo('inventory')">go here</button>
+        some info about admin inventory stuff
+      </div>
+    </div>
+  </ng-template>
+</div> -->
```

Comment:
why leave commented code?

Reply:
us1 did not work in this module. We do not feel comfortable taking out code that is not ours.